internal/gengapic/testdata/deprecated_client_init.want

```diff
@@ -0,0 +1,121 @@
+// internalClient is an interface that defines the methods availaible from Awesome Foo API.
+type internalClient interface {
+	Close() error
+	setGoogleClientInfo(...string)
+	Connection() *grpc.ClientConn
+	Zip(context.Context, *mypackagepb.Bar, ...gax.CallOption) (*mypackagepb.Foo, error)
+}
+
+// Client is a client for interacting with Awesome Foo API.
+// Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.
+//
+// Foo service does stuff.
+// Foo is deprecated.
```

Comment:
Per my previous comments, this looks a little odd having the added "Foo is deprecated" statement, I think the Deprecation tag is sufficient.